content/en/logs/log_collection/csharp.md

```diff
@@ -260,9 +260,13 @@ If, despite the benefits of logging in JSON, you wish to log in raw string forma
 {{% /tab %}}
 {{< /tabs >}}
 
-**Connect Logs and Traces**
+## Connect your service across Logs and APM
```

Comment:
I believe we should keep the word `traces`.

We are not saying Log Management here as we are talking about the data points, not the products. So we are connecting the traces and the logs, not Log Management and APM.